blockly-rtc/src/index.js

```diff
@@ -22,7 +22,7 @@
  */
 
 import * as Blockly from 'blockly';
-import {getEvents, writeEvents} from './api';
```

Comment:
Quick question since I'm out of date: where is http_handlers referenced now?

Reply:
http_handlers is currently not reference right now. WorkspaceClient can be initiated with either http_handlers or websocket_handlers and we are using websocket_handlers as the default for now.